features/FEATURE_BLE/ble/BLEProtocol.h

```diff
@@ -59,18 +59,51 @@ namespace BLEProtocol {
              */
             PUBLIC = 0,
 
+            /**
+             * Random address.
+             *
+             * Use Gap::getRandomAddressType to retrieve the type of the random
+             * address.
+             */
+            RANDOM,
```

Comment:
That's a good question and I don't have a proper answer, I started by adding the new values at the end but realized it would break existing implementation where the value is directly passed from the stack. 
It should be fine to add these at the end if all the vendor implementations are fixed first.